### DISCUSSION TITLE
Version of Edge and Google Chrome on Windows 2022

### DISCUSSION THREAD

**Discussion by vamshisiram** [actions/virtual-environments#7800] [General questions]
What version of Google Chrome and MS Edge browsers are installed on a Windows Server 2022 image?

Is it documented in the code somewhere, like in a json file. So, i can see it?

@Alexey-Ayupov @sergey-akhalkov @mikhailkoliada  sorry for tagging you, looking to get quick answers if possible.

**Comment by kmrozConfigit** [comment]
e.g. https://github.com/actions/runner-images/blob/win22/20230820.1/images/win/Windows2022-Readme.md
